content/applications/inventory_and_mrp/plm/manage_changes/eco_type.rst

```diff
@@ -5,22 +5,37 @@
 .. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
 .. |ECOs| replace:: :abbr:`ECOs (Engineering Change Orders)`
 
-========
-ECO type
-========
+====================
+ECO types and stages
+====================
 
-An *ECO type* is assigned to *engineering change orders* (ECOs) to organize and track changes to
-products and bills of materials (BoMs). Each |ECO| type separates |ECOs| into a project in Gantt
-view, ensuring collaborators and stakeholders **only** view and assist with relevant |BOM|
-improvements.
+Engineering change orders (ECOs) are categorized by the type of change they represent and their
+statuses are tracked via stages. Both |ECO| types and stages are defined by the user and can be
+customized to the specific needs of a business or industry.
 
-For example, an electronic chip manufacturer might use 'New Product Introduction', 'Product
-Improvement', 'Component Change', and 'Firmware Update' |ECO| types. Then, designers and engineers
-can focus on |ECOs| in the 'New Product Introduction' and 'Product Improvement' projects, avoiding
-unrelated supplier change or firmware update |ECOs|.
+ECO types
+=========
 
-Create ECO type
-===============
+*Engineering change orders* (ECOs) require an |ECO| type to organize and track changes to products
+and bills of materials (BoMs). Like a :doc:`product
+type<../../inventory/product_management/configure/type>`, |ECO| types are highly customizable and
+specific to the needs of individual businesses. Each |ECO| type separates |ECOs| into different
+projects in the :guilabel:`PLM Overview`, ensuring collaborators and stakeholders only view and
+assist with relevant |BOM| improvements.
+
+.. example::
```

Comment:
Good example but could be simpler and more relevant to common use-cases (instead of changing defaults, could just add one new one to indicate that the business is scaling its engineering ops, more likely and easier to understand.)